hypervisor/arch/x86/boot/cpu_primary.S

```diff
@@ -162,7 +162,7 @@ after:
     mov     %eax,%gs  // Was 32bit POC CLS
 
    /* continue with chipset level initialization */
-   call     bsp_boot_init
+   call     init_boot_cpu
```

Comment:
We already has API of init_secondary_cpu.
TO match it, how about init_primary_cpu?

Reply:
ok, will update.